client/src/components/SidePanel/SidePanel.tsx

```diff
@@ -173,7 +173,8 @@ const SidePanel = ({
           <ResizablePanel
             tagName="nav"
             id="controls-nav"
-            aria-label="controls-nav"
+            aria-label="Controls"
```

Comment:
Ideally, we use localizations for aria labels.

Refer to the base i18n language file:

https://github.com/danny-avila/LibreChat/blob/02847af58033f024daf53d7674cb0a16ae518dfa/client/src/localization/languages/Eng.ts#L89

Also, how these keys are used app-wide:
https://github.com/danny-avila/LibreChat/blob/02847af58033f024daf53d7674cb0a16ae518dfa/client/src/components/Nav/NewChat.tsx#L93

Some of the initial aria labels I made are missing this, and will be updated, so help in this regard is appreciated.